test/utils/handleRangeHeaders.test.js

```diff
@@ -0,0 +1,75 @@
+import handleRangeHeaders from '../../src/utils/handleRangeHeaders';
+
+describe('handleRangeHeaders', () => {
+  it('should return content in range with valid range header', () => {
+    const content = 'abcdef';
+    const req = {
+      headers: {
+        range: 'bytes=1-4',
+      },
+    };
+
+    const res = {
+      setHeader: jest.fn(),
+    };
+
+    const contentRes = handleRangeHeaders(content, req, res);
+    expect(contentRes).toEqual('bcde');
+    expect(res.statusCode).toEqual(206);
+    expect(res.setHeader.mock.calls).toMatchSnapshot();
+  });
+
+  it('should handle malformed range header', () => {
+    const content = 'abcdef';
+    const req = {
+      headers: {
+        range: 'abc',
+      },
+    };
+
+    const res = {
+      setHeader: jest.fn(),
+    };
+
+    const contentRes = handleRangeHeaders(content, req, res);
+    expect(contentRes).toEqual('abcdef');
+    expect(res.statusCode).toBeUndefined();
```

Comment:
@evilebottnawi Is this expected behavior that the statusCode is not set to something on malformed range header?

Reply:
We need looks in docs/specs, i think statusCode should always be

Reply:
I think it is ok to ignore malformed range, log error in console and return original code, i.e. `200` if no problems before